source/crud/tests/v2/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,50 @@
+runOn:
+  -
+    # arrayFilters support first introduced in 3.5.6
+    maxServerVersion: "3.5.5"
+
+data:
+  - {_id: 1, y: [{b: 3}, {b: 1}]}
+  - {_id: 2, y: [{b: 0}, {b: 1}]}
+
+tests:
+  -
+    description: "BulkWrite on server that doesn't support arrayFilters"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              # UpdateOne with with arrayFilters
+              name: "updateOne"
+              arguments:
+                filter: {}
+                update: { $set: { "y.0.b": 2 } }
+                arrayFilters: [ { "i.b": 1 } ]
+          options: { ordered: true }
+          error: true
```

Comment:
I missed this in the first review, but `error` should appear at the top level of the `operation` element. This has it placed within `arguments`.

Reply:
I missed that as well.  Auto-indent probably "helped" me out there.  Fixed.